### DISCUSSION TITLE
how do i change email in "contact aa near me" section of app?

### DISCUSSION THREAD

**Discussion by ljmorrone** [code4recovery/12-step-meeting-list#828] [Q&A] [answered]
Hi, 

The section of the app for "Contact AA Near Me" lists our (Albuquerque NM) central office's email for the web editor, but we want the emails to go to the schedule editor at a different email address. The web editor is receiving emails from people using this section of the app, but lacks the context to answer them fully, and the emails have to be forwarded to the schedule editor anyway. It would save us a great deal of trouble by cutting down on unnecessary email labor to get this email addressed switched, but I can't figure out how to change it. 

How do I change the email in the "Contact AA Near Me" section of the Meeting Guide App?

Thanks!

**Comment by joshreisner** [comment, marked as answer]
just reach out to appsupport@aa.org, they can help you out!
